.github/workflows/pdf-visual-qa.yml

```
@@ -0,0 +1,198 @@
+          visual_qa_page_*_annotated.png
+          diff_page_*.png
+        retention-days: 30
+        
+    - name: Comment PR with Results
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const fs = require('fs');
+          const glob = require('glob');
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)